src/shogun/base/init.cpp

```diff
@@ -191,12 +201,20 @@ namespace shogun
 		SG_REF(sg_rand);
 		return sg_rand;
 	}
+
+	CSignal* get_global_signal()
```

Comment:
this as well could be part of `#ifndef SWIG`?
do we wanna expose signals to swig interfaces? @karlnapf @lisitsyn